exercises/practice/leap/.approaches/case/snippet.txt

```diff
@@ -0,0 +1,8 @@
+def leap_year?(year) do
+    case { rem(year, 400), rem(year, 100), rem(year, 4) } do
+        { 0, _, _ } -> true
+        { _, 0, _ } -> false
+        { _, _, 0 } -> true
+        { _, _, _ } -> false
```

Comment:
Since this branch represents "all else", I feel a more idiomatic way to write it would be `_ -> false` since the shape of the tuple is not important anymore.

```suggestion
        _ -> false
```

Reply:
It's something to discuss in the details of the approach, I think. I hope we will be able to discuss this once I have written up the approach.

Reply:
Still, this snippet should use the more idiomatic version, which is the version @Tuxified suggested